Tasks/PipAuthenticateV0/package.json

```diff
@@ -18,9 +18,9 @@
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
         "@types/mocha": "^5.2.7",
-        "@types/node": "^10.17.0",
+        "@types/node": "^16.11.39",
         "@types/uuid": "^8.3.0",
-        "azure-pipelines-task-lib": "^3.1.7",
+        "azure-pipelines-task-lib": "^4.0.0-preview",
```

Comment:
We have a newer package version resolving several vulnerabilities.
Can we use this version? - https://www.npmjs.com/package/azure-pipelines-task-lib/v/4.0.2

Reply:
updated to 4.1.0